api/src/test/java/org/pmiops/workbench/api/DataSetControllerTest.java

```diff
@@ -771,23 +755,12 @@ public void testGetRQuery() {
                 + "AND (c_occurrence.PERSON_ID IN (SELECT * FROM person_id from `"
                 + TEST_CDR_TABLE
                 + ".person` person)) \n"
-                + "LIMIT \", max_number_of_rows)\n"
-                + "\n"
-                + prefix
-                + "query_config <- list(\n"
-                + "  query = list(\n"
-                + "    parameterMode = 'NAMED',\n"
-                + "    queryParameters = list(\n\n"
-                + "    )\n"
-                + "  )\n"
-                + ")\n"
+                + "LIMIT \", max_number_of_rows, sep=\"\")\n"
```

Comment:
We need to have test cases for all the edge cases,  like when things that are expected to be there are missing. We should be failing the operation early instead of sending a broken query to the notebook if anything is amiss. (We don't have a great automatic way to find all the notebooks with queries for a given release yet from what I remember.)